packages/cdktf-cli/bin/cmds/ui/terraform-json.ts

```diff
@@ -0,0 +1,31 @@
+export interface TerraformJsonMetadata {
+  version: string;
+  stackName: string;
+}
+export interface TerraformJsonRootComment {
+  metadata: TerraformJsonMetadata;
+}
+
+export interface TerraformJsonConfigBackendRemoteWorkspace {
+  name?: string;
+  prefix?: string;
+}
+
+export interface TerraformJsonConfigBackendRemote {
+  organization: string;
+  hostname?: string;
+  token?: string;
+  workspaces: TerraformJsonConfigBackendRemoteWorkspace;
+}
+
+export interface TerraformJsonConfigBackend {
+  remote?: TerraformJsonConfigBackendRemote;
+}
+export interface TerraformJsonConfig {
+  backend?: TerraformJsonConfigBackend;
+}
+export interface TerraformJson {
```

Comment:
Would be nice to reuse something from the core library for this.

Reply:
Yes 👍  #492